app/src/main/java/org/jellyfin/androidtv/ui/playback/nextup/NextUpViewModel.kt

```diff
@@ -54,20 +59,35 @@ class NextUpViewModel(
 	private suspend fun loadItemData(id: String) = withContext(Dispatchers.IO) {
 		val item = apiClient.getItem(id) ?: return@withContext null
 
-		val thumbnail = apiClient.GetImageUrl(item, ImageOptions())
+		val thumbnail = when (userPreferences[UserPreferences.nextUpBehavior] == NextUpBehavior.EXTENDED) {
+			true -> apiClient.GetImageUrl(item, ImageOptions())
+			false -> null
+		}
 		val logo = apiClient.GetLogoImageUrl(item, ImageOptions())
 
-		val title = if (item.indexNumber != null && item.name != null)
-			"${item.indexNumber}. ${item.name}"
-		else if (item.name != null)
-			item.name
-		else ""
+		val seasonNumber = when (item.parentIndexNumber != null && item.parentIndexNumber != 0) {
+			true -> context.getString(R.string.lbl_season_number, item.parentIndexNumber)
+			false -> null
+		}
+		val episodeNumber = when {
+			(item.parentIndexNumber == 0) ->
+				context.getString(R.string.lbl_special)
+			(item.indexNumber != null) ->
+				listOfNotNull(
+					context.getString(R.string.lbl_episode_number, item.indexNumber),
+					item.indexNumberEnd?.toString()
+				).joinToString("–")
```

Comment:
It might be better to use 2 string resources. something like "lbl_episode_number" and "lbl_episode_range". That way translators can adapt it to their language. (some languages might use a different order etc.)